client/my-sites/people/team-invite/style.scss

```diff
@@ -42,6 +42,9 @@
 		margin-bottom: 1rem;
 	}
 }
+.team-invite-form-learn-more__link {
+	color: var(--color-accent);
```

Comment:
Should this be `—color-link`? If I recall correctly, it’s slightly different on a few schemes like Sakura.

```suggestion
	color: var(--color-link);
```

Reply:
Thank you, @Aurorum, I guess you're absolutely right :)